### ISSUE TITLE
Add Rogue Metric

### ISSUE THREAD

**Issue by PetrochukM** [PetrochukM/PyTorch-NLP#6]
Tensor2Tensor Implementation: https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/utils/rouge.py
